src/features/canvass/utils/sortMetrics.ts

```diff
@@ -0,0 +1,11 @@
+import { ZetkinMetric } from 'features/areaAssignments/types';
+
+export default function sortMetrics(
+  metricsList: ZetkinMetric[]
+): ZetkinMetric[] {
+  return metricsList
+    ?.slice()
```

Comment:
Is the purpose of the `slice()` to create a copy of the array? I've never seen that done before, but it makes sense. The more common(?) way to do it is to use `concat()`, probably because it's older, but I guess `slice()` is fine to. I would maybe create a variable for it though, to be clear that that's what it's doing:

```
const listCopy = metricsList.slice(); // or .concat()
return metricsList.sort(
```

Reply:
Good idea!